File: php_SDK/com/payhub/ws/api/TransactionType.php
<?php

/**
 * Created by PhpStorm.
 * User: agustin
 * Date: 24/07/2015
 * Time: 09:11
 */
class TransactionType
{
    const Sale="forSale";
    const AuthOnly="forAuthOnly";
    const Capture="forCapture";
    const Bill="forBill";
    const CardData="forCardData";
    const Customer="forCustomer";
    const Merchant="forMerchant";
    const RecurringBill="forRecurringBill";
    const Schedule="forSchedule";
    const Refund="forRefund";
    const VoidTransaction="forVoid";
    const Verify="forVerify";
}

File: Payhub - SDK/java_SDK/PayHubSDK/src/main/java/com/payhub/ws/api/TransactionType.java
/**
 * 
 */
package com.payhub.ws.api;

/**
 * @author agustin
 *
 */
public enum TransactionType {
	Sale,//forSale
	AuthOnly,//forAuthOnly
	Capture,//forCapture
	Bill,//forBill
	CardData,//forCardData
	Customer,//forCustomer
	Merchant,//forMerchant
	RecurringBill,//forRecurringBill
	Schedule,//forSchedule
	Refund,//forRefund
	VoidTransaction,
	Verify//forVoid
}
